scripts/talos/README.md

```diff
@@ -0,0 +1,53 @@
+# Talos
+
+## Description
+
+Talos is a bumper bot that runs automatically for Psammead packages.
+
+## Manual Usage
+
+### Using Jenkins
+
+Navigate to [Jenkins](https://ci.news.tools.bbc.co.uk/blue/organizations/jenkins/psammead/branches) and click on the play (run) button, then enter the list of packages you want to bump seperated by commas.
+
+![invoke-talos-jenkins](https://user-images.githubusercontent.com/34196381/63758525-d03ef980-c8b3-11e9-9b8c-d4f9a451237b.png)
```

Comment:
This is updating the alt text for the image so that it is descriptive. 
```suggestion
![screenshot showing the Jenkins interface, starting a Talos job updating react,styled-components](https://user-images.githubusercontent.com/34196381/63758525-d03ef980-c8b3-11e9-9b8c-d4f9a451237b.png)
```